Tests/SwiftDocCTests/Rendering/DeclarationsRenderSectionTests.swift

```diff
@@ -52,7 +63,8 @@ class DeclarationsRenderSectionTests: XCTestCase {
                     DeclarationRenderSection(
                         languages: nil,
                         platforms: [],
-                        tokens: [.init(text: "", kind: token)]
+                        tokens: [.init(text: "", kind: token)],
+                        otherDeclarations: [DeclarationRenderSection.OtherDeclaration(tokens: [.init(text: "", kind: token)], identifier: "identifier")]
```

Comment:
Maybe we should also have a test that shows that we _don't_ emit an `otherDeclarations` section if this is empty?